pkg/test/utils/kubernetes.go

```diff
@@ -1083,7 +1109,7 @@ func RunCommands(logger Logger, namespace string, commands []harness.Command, wo
 
 	for i, cmd := range commands {
 
-		bg, err := RunCommand(context.Background(), namespace, cmd, workdir, logger, logger, logger, timeout)
+		bg, err := RunCommand(ctx, namespace, cmd, workdir, logger, logger, logger, timeout)
```

Comment:
just wondering what motivated this change? Was it that originally you planned to pass in context from the top but in the end did not end up doing it and considered it out of scope of this PR?

Reply:
Yes, I was preparing to do bigger changes on the timeouts (as you suggested) but the other conversations in the ticket seemed that it would be too big of a change, so I left what made sense on it's own and was a good golang practice (pass the context).
NExt week I could give another hand for the global timeouts if you would like.

Reply:
Sounds great, thank you :)